package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -0,0 +1,702 @@
+                           <DOUBLE_T*> &c_box[0], self.charmm)
+        if ok != 0:
+            raise IOError("Couldn't write DCD frame: reason {}".format(DCD_ERRORS[ok]))
+
+        self.current_frame += 1
+
+    def read(self):
+        """
+        Read next dcd frame
+
+        Returns
+        -------
+        DCDFrame : namedtuple
+            positions are in ``x`` and unitcell in ``unitcell`` attribute of DCDFrame
+
+        Notes
+        -----
+        unitcell is read as is from DCD. Post processing depending on the program this
+        DCD file was written with is necessary. Have a look at the MDAnalysis DCD reader
+        for possible post processing into a common unitcell data structure.
+
+        Raises
+        ------
+        IOError
+        StopIteration
+        """
+        if self.reached_eof:
+            raise IOError('Reached last frame in DCD, seek to 0')
+        if not self.is_open:
+            raise IOError("No file open")
+        if self.mode != 'r':
+            raise IOError('File opened in mode: {}. Reading only allow '
+                               'in mode "r"'.format('self.mode'))
+        if self.n_frames == 0:
+            raise IOError("opened empty file. No frames are saved")
+
+        cdef np.ndarray xyz = np.empty((self.natoms, self.ndims), dtype=FLOAT, order='F')
+        cdef np.ndarray unitcell = np.empty(6, dtype=DOUBLE)
+        unitcell[0] = unitcell[2] = unitcell[5] = 0.0;
+        unitcell[4] = unitcell[3] = unitcell[1] = 90.0;
+
+        first_frame = self.current_frame == 0
+        ok = self.c_readframes_helper(xyz[:, 0], xyz[:, 1], xyz[:, 2], unitcell, first_frame)
+        if ok != 0 and ok != -4:
+            raise IOError("Reading DCD header failed: {}".format(DCD_ERRORS[ok]))
+
+        # we couldn't read any more frames.
+        if ok == -4:
+            self.reached_eof = True
+            raise StopIteration
+
+        self.current_frame += 1
+        return DCDFrame(xyz, unitcell)
+
+
+    def readframes(self, start=None, stop=None, step=None, order='fac', indices=None):
+        """
+        read multiple frames at once
+
+        Parameters
+        ----------
+        start : int (optional)
+            starting frame, default to 0
+        stop : int (optional)
+            stop frame, default to ``n_frames``
+        step : int (optional)
+            step between frames read, defaults to 1
+        order : str (optional)
+            give order of returned array with `f`:frames, `a`:atoms, `c`:coordinates
+        indices : array_like (optional)
+            only read selected atoms. In ``None`` read all.
+
+        Returns
+        -------
+        DCDFrame : namedtuple
+            positions are in ``x`` and unitcell in ``unitcell`` attribute of DCDFrame.
+            Here the attributes contain the positions for all frames in the given order
+
+        Notes
+        -----
+        unitcell is read as it from DCD. Post processing depending the program this
```

Comment:
I don't quite follow your reasoning here. Why would adding more information make it more difficult to make it its own package??
It's not even an internal link to the docs but to a place with notes on the DCD file format. I happens to be the MDAnalysis wiki, but we are also linking to the NAMD docs and wherever else. Given the amount of discussion and uncertainty about everything even among developers I am pretty sure that anyone using it will appreciate a link towards some of the existing knowledge.